### ISSUE TITLE
[BUG REPORT]

### ISSUE THREAD

**Issue by adam248** [bitfinexcom/bfx-report-electron#109]
An error occurred

```vim
Cannot download differentially, fallback to full download: Error: ENOENT: no such file or directory, open 'C:/Users/Adam/AppData/Local/bfx-report-electron-updater/installer.exe'
```

---

| Debug info | |
| :--- | :--- |
| Version | 3.5.2 |
| Commit Hash | 236a7bdd63077f876a24e925ae0698ce9c30a6f4 |
| Commit Date | 2021-08-09T10:41:05Z |
| Electron | 13.1.7 |
| Chrome | 91.0.4472.124 |
| Node.js | 14.16.0 |
| V8 | 9.1.269.36-electron.0 |
| OS version | Windows 10 Home |
| OS release | Windows_NT x64 10.0.19042 |
| CPUs | AMD Ryzen 7 1800X Eight-Core Processor          x 16 |
| RAM | 63.95GB (39.65GB free) |
| App RAM limit | 4095.75Mb (9.5MB used) |

<details>

<summary>Main log</summary>

```vim
[2021-08-25 09:24:30.262] [error] Cannot download differentially, fallback to full download: Error: ENOENT: no such file or directory, open 'C:/Users/Adam/AppData/Local/bfx-report-electron-updater/installer.exe'

```

</details>

<details>

<summary>Worker errors</summary>

```vim
WORKER:ERROR [2021-08-24T02:44:38.801Z] 	
  - METHOD_NAME: getFullSnapshotReport [PROTECTED]
  - DataConsistencyError: ERR_COLLECTIONS_DATA_IS_NOT_CONSISTENT
    at DataConsistencyChecker.check (D:\Bitfinex Work Files\BitfinexReport-3.5.2-x64-win\resources\app\bfx-reports-framework\workers\loc

```

</details>

<details>

<summary>Worker exceptions</summary>

```vim
Empty
```

</details>

---

*The issue was opened from the electron app*

**Comment by adam248** [comment]
The fallback worked just for the record. Thanks

**Comment by ezewer** [comment]
Thanks for reporting, would check!

**Comment by ezewer** [comment]
@adam248 this has been fixed on last version 3.5.5 please re open if still have this issue with the last version.
